test/runtime/samples/style_manager-cleanup/_config.js

```diff
@@ -0,0 +1,18 @@
+export default {
+	skip_if_ssr: true,
+	skip_if_hydrate: true,
+	skip_if_hydrate_from_ssr: true,
```

Comment:
Oh, i added them because i thought it wouldn't be necessary to test in these cases. Reason being that you only really trigger transition in the browser on runtime. If this is wrong then please let me know! @baseballyama

Reply:
I asked this because the other tests with transitions did not have these three lines.
But this is logically right!